gms/client/src/main/java/com/linkedin/dataprocess/client/DataProcesses.java

```diff
@@ -0,0 +1,104 @@
+    private static final DataProcessInfoRequestBuilders PROCESS_INFO_REQUEST_BUILDERS = new DataProcessInfoRequestBuilders();
+
+    protected DataProcesses(@Nonnull Client restliClient) {
+        super(restliClient);
+    }
+
+    @Nonnull
+    public DataProcess get(@Nonnull DataProcessUrn urn)
+            throws RemoteInvocationException {
+        GetRequest<DataProcess> getRequest = PROCESSES_REQUEST_BUILDERS.get()
+                .id(new ComplexResourceKey<>(toDataProcessKey(urn), new EmptyRecord()))
+                .build();
+
+        return _client.sendRequest(getRequest).getResponse().getEntity();
+    }
+
+    @Nonnull
+    public Map<DataProcessUrn, DataProcess> batchGet(@Nonnull Set<DataProcessUrn> urns)
+            throws RemoteInvocationException {
+        BatchGetEntityRequest<ComplexResourceKey<DataProcessKey, EmptyRecord>, DataProcess> batchGetRequest
+                = PROCESSES_REQUEST_BUILDERS.batchGet()
+                .ids(urns.stream().map(this::getKeyFromUrn).collect(Collectors.toSet()))
+                .build();
+
+        return _client.sendRequest(batchGetRequest).getResponseEntity().getResults()
+                .entrySet().stream().collect(Collectors.toMap(
+                        entry -> getUrnFromKey(entry.getKey()),
+                        entry -> entry.getValue().getEntity())
+                );
+    }
+
+    public void createDataProcessInfo(@Nonnull DataProcessUrn dataProcessUrn,
+                                   @Nonnull DataProcessInfo dataProcessInfo) throws RemoteInvocationException {
+        CreateIdRequest<Long, DataProcessInfo> request = PROCESS_INFO_REQUEST_BUILDERS.create()
+                .dataprocessKey(new ComplexResourceKey<>(toDataProcessKey(dataProcessUrn), new EmptyRecord()))
+                .input(dataProcessInfo)
+                .build();
+        _client.sendRequest(request).getResponse();
+    }
+
+    @Nonnull
+    private DataProcessKey toDataProcessKey(@Nonnull DataProcessUrn urn) {
+        return new DataProcessKey().setName(urn.getNameEntity());
+    }
+
+    @Nonnull
+    protected DataProcessUrn toDataProcessUrn(@Nonnull DataProcessKey key) {
+        return new DataProcessUrn(key.getOrchestrator(), key.getName(), key.getOrigin());
+    }
+
+    @Nonnull
+    private ComplexResourceKey<DataProcessKey, EmptyRecord> getKeyFromUrn(@Nonnull DataProcessUrn urn) {
+        return new ComplexResourceKey<>(toDataProcessKey(urn), new EmptyRecord());
+    }
+
+    @Nonnull
+    private DataProcessUrn getUrnFromKey(@Nonnull ComplexResourceKey<DataProcessKey, EmptyRecord> key) {
+        return toDataProcessUrn(key.getKey());
+    }
+
+    @Nonnull
+    @Override
+    public CollectionResponse<DataProcess> search(@Nonnull String input, @Nullable Map<String, String> requestFilters, int start, int count)
+            throws RemoteInvocationException {
+        return null;
```

Comment:
throw UnsupportedOperationException exception instead?

Reply:
I have planed release this feature in the next PR.  New code has included this UnsupportedOperationException